app/code/community/Kirchbergerknorr/UrlCacheControl/etc/config.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * @author      Nick Dilßner <nd@kirchbergerknorr.de>
+ * @copyright   Copyright (c) 2017 kirchbergerknorr GmbH (http://www.kirchbergerknorr.de)
+ * @license     http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+-->
+
+<config>
+    <modules>
+        <kirchbergerknorr_urlCacheControl>
```

Comment:
Please consider capitalising the k and u, youre module path is capitalised as well.

Reply:
+1